types/amphtml-validator/validator.proto.d.ts

```diff
@@ -0,0 +1,156 @@
+/**
```

Comment:
This file contains only type definitions -- might be better to just include all of this in `index.d.ts`.

Reply:
I've consolidated the extra type definitions all into `index.d.ts`.